sdk/identity/Azure.Identity/tests/InteractiveBrowserCredentialTests.cs

```diff
@@ -294,25 +294,25 @@ public async Task InvokesBeforeBuildClientOnExtendedOptions()
         }
 
         [Test]
-        public async Task BrowserCustomizedOptionsHtmlMessage([Values(null, "<p> Login Successfully.</p>")] string htmlMessageSuccess, [Values(null, "<p> An error occured: {0}. Details {1}</p>")] string htmlMessageError)
+        public async Task BrowserCustomizationsHtmlMessage([Values(null, "<p> Login Successfully.</p>")] string htmlMessageSuccess, [Values(null, "<p> An error occured: {0}. Details {1}</p>")] string htmlMessageError)
```

Comment:
Nit: Should "Customizations" be singular instead?

Reply:
Probably, but shouldn't matter for a test.